Added filter by brand in map view.

mribeiro opened the pull request:
Brands shown are the ones using VOST API to report fuel availability.
List of brands is hardcoded in map.js file.

The result is expected to be the same as the gif below.
![2019-08-15 17-24-18 2019-08-15 17_27_41](https://user-images.githubusercontent.com/1676406/63112215-94d52e80-bf87-11e9-8ed4-319ee8b400d1.gif)

Resolves #156

Commit:
Added filter by brand in map view.
Brands shown are the ones using VOST API to report fuel availability.
List of brands is hardcoded in map.js file.

miguelantoniosantos committed:
Merge branch 'master' into GTP-156/map-filter-by-brand

miguelantoniosantos reviewed (commented):
Can you run ```npm run prod```?

Commit:
Fixed brand filter reset not working

Instead of the html anchor calling a JS function, there's a click handler in map.js (minification process was killing this).
Also uploading the result of npm run prod .
